youtube_dl/YoutubeDL.py

```diff
@@ -848,6 +849,23 @@ def iter_playlistitems(format):
                 random.shuffle(entries)
 
             for i, entry in enumerate(entries, 1):
+                if self.params.get('playlistprompt', False):
+                    if 'title' in entry:
+                        skipnextvid = False
+                        while True:
+                            self.to_screen('Do you want to download video %s/%s: "%s"? (y/n) ' %
```

Comment:
It's supposed to just ask again if you don't decide for y or n. Should I change it so that yes is default?